<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>translatorClass</class>
 <widget class="QMainWindow" name="translatorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>translator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="EnterWindow">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>531</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="ResultWindow">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>531</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="TranslateButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Перевести</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="RomeNumber">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>531</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Арабские цифры</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Римские цифры</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="translator.qrc"/>
 </resources>
 <connections/>
</ui>
